projects/igniteui-angular/src/lib/splitter/splitter.component.ts

```diff
@@ -175,6 +175,19 @@ export class IgxSplitterComponent implements AfterContentInit {
         this.panes?.notifyOnChanges();
     }
 
+    /**
+     * Sets the visibility of the handle and expanders in the splitter bar.
+     * True by default
+     * 
+     * @example
+     * ```html
+     * <igx-splitter [showControls]='false''>
+     * </igx-splitter>
+     * ```
+     */
+    @Input()
+    public showControls: boolean = true; // Input to toggle showing/hiding controls
```

Comment:
I am not so sure about the name. Perhaps something like `collapsible` can be a better naming as it more widely captures what is being disabled/enabled.